homeassistant/components/google_tasks/config_flow.py

```diff
@@ -28,3 +35,24 @@ def extra_authorize_data(self) -> dict[str, Any]:
             "access_type": "offline",
             "prompt": "consent",
         }
+
+    async def async_oauth_create_entry(self, data: dict[str, Any]) -> FlowResult:
+        """Create an entry for the flow."""
+        try:
+            resource = build(
+                "tasks",
+                "v1",
+                credentials=Credentials(token=data[CONF_TOKEN][CONF_ACCESS_TOKEN]),
+            )
+            cmd: HttpRequest = resource.tasklists().list()
+            await self.hass.async_add_executor_job(cmd.execute)
+        except HttpError as ex:
+            error = ex.reason
+            return self.async_abort(
+                reason="access_not_configured",
+                description_placeholders={"message": error},
+            )
+        except Exception as ex:  # pylint: disable=broad-except
+            self.logger.error("Unknown error occurred: %s", ex.args)
```

Comment:
We normally include the stack trace for unknown exceptions.

Reply:
Does it work if I change error to `exception`?

Reply:
```suggestion
            self.logger.exception("Unknown error occurred: %s", ex)
```